kubernetes-manifests/client.deployment.yaml

```diff
@@ -41,6 +41,8 @@ spec:
           - "--qps=${qps}"
           - "--rpc=${rpc}"
           - "--metadata=${metadata}"
+          - "--request_payload_size=${request_payload_size}"
```

Comment:
These should be behind an if , similar to `% if enable_csm_observability:`

Reply:
Fixed.